com.unity.netcode.adapter.utp/Runtime/NetworkMetricsPipelineStage.cs

```diff
@@ -0,0 +1,53 @@
+#if MULTIPLAYER_TOOLS
+using AOT;
+using Unity.Burst;
+using Unity.Collections.LowLevel.Unsafe;
+using Unity.Networking.Transport;
+using UnityEngine;
+
+namespace Unity.Netcode
+{
+    [BurstCompile]
+    public unsafe struct NetworkMetricsPipelineStage : INetworkPipelineStage
+    {
+        static TransportFunctionPointer<NetworkPipelineStage.ReceiveDelegate> ReceiveFunction = new TransportFunctionPointer<NetworkPipelineStage.ReceiveDelegate>(Receive);
+        static TransportFunctionPointer<NetworkPipelineStage.SendDelegate> SendFunction = new TransportFunctionPointer<NetworkPipelineStage.SendDelegate>(Send);
+        static TransportFunctionPointer<NetworkPipelineStage.InitializeConnectionDelegate> InitializeConnectionFunction = new TransportFunctionPointer<NetworkPipelineStage.InitializeConnectionDelegate>(InitializeConnection);
+
+        public NetworkPipelineStage StaticInitialize(byte* staticInstanceBuffer, int staticInstanceBufferLength, NetworkSettings settings)
+        {
+            return new NetworkPipelineStage(ReceiveFunction,
```

Comment:
Nit: could you wrap these parameters up according to our coding standard? For reference:
> When wrapping params, do not leave any on line with function name
> When wrapping, only indent one stop (do not line up with paren)